tests/foreman/ui/test_repository.py

```diff
@@ -1199,3 +1222,38 @@ def test_positive_drpm_sync_publish_promote_cv(self):
         )
         self.assertEqual(result.return_code, 0)
         self.assertGreaterEqual(len(result.stdout), 1)
+
+    @run_in_one_thread
+    @tier2
+    def test_positive_reposet_disable(self):
+        """Enable RH repo, sync it and then disable
+
+        @id: de596c56-1327-49e8-86d5-a1ab907f26aa
+
+        @Assert: RH repo was disabled
+
+        @CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        with manifests.clone() as manifest:
+            upload_manifest(org.id, manifest.content)
+        repos = self.sync.create_repos_tree(SAT6_TOOLS_TREE)
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(org.name)
+            session.nav.go_to_red_hat_repositories()
+            # Enable RH repository
+            self.sync.enable_rh_repos(repos, REPO_TAB['rpms'])
+            session.nav.go_to_sync_status()
+            # Sync the repo and verify sync was successful
+            self.assertTrue(self.sync.sync_noversion_rh_repos(
+                PRDS['rhel'], [REPOS['rhst6']['name']]
+            ))
+            session.nav.go_to_red_hat_repositories()
+            # Click the checkbox once more to disable the repo
+            self.sync.enable_rh_repos(repos, REPO_TAB['rpms'])
```

Comment:
I left a comment above this line that we are using mentioned method once more to click again on that checkbox and as a result it will become disabled. The thing is i don't think it's a good idea to create `disable_rh_repos` helper which will have absolutely the same implementation. 
Rename it to something different? First, it will affect other tests using this helper, second, something like `switch_rh_repo_status` won't be that clear in all 99% of the rest calls, as all of them are enabling repo.